common/supplemental/units.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE supplementalData SYSTEM "../../common/dtd/ldmlSupplemental.dtd">
+<!--
+Copyright © 1991-2013 Unicode, Inc.
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+For terms of use, see http://www.unicode.org/copyright.html
+-->
+
+<supplementalData>
+	<version number="$Revision$"/>
+	<convertUnits>
+		<convertUnit source='g-force' target='meter-per-second-squared' factor='9.80665' />
+		<convertUnit source='meter-per-second-squared' target='meter-per-second-squared' factor='1' />
+		<convertUnit source='arc-minute' target='degree' factor='0.016666666666666666' />
+		<convertUnit source='arc-second' target='degree' factor='0.0002777777777777778' />
+		<convertUnit source='degree' target='degree' factor='1' />
+		<convertUnit source='radian' target='degree' factor='0.01745329' />
```

Comment:
This is an example where the conversion is an irrational number.  Do we have a standard for how many significant figures we want in such cases?